components/x-podcast-launchers/src/generate-rss-url.js

```diff
@@ -0,0 +1,8 @@
+import nanoid from 'nanoid';
+
+const ACAST_RSS_URL = 'https://access.acast.cloud/rss';
+
+export default function generateRSSUrl(seriesId) {
+	const token = nanoid(10);
```

Comment:
Would it be worth logging these to spoor, just so in the future we'd have a list of all the tokens we'd issued?

Reply:
Part of me thinks we should either store them properly or not store them at all... where would they actually go if we log them in spoor?

I suppose also we would have the o-tracking stuff on the hrefs on the links themselves?

Reply:
We have an easy way to revoke these with Acast and Matt (PO) doesn't need them